src/plugins/kibana/public/settings/sections/indices/styles/_add_data_wizard.less

```diff
@@ -0,0 +1,88 @@
+@import (reference) "~ui/styles/variables";
+@import (reference) "~ui/styles/mixins";
+@import (reference) "~ui/styles/theme";
+
+.wizard-container {
+  padding: 0 10px;
+
+  .form-group {
+    margin-bottom: 5px;
+  }
+
+  .form-control {
+    background-color: #f5fafb;
+    border-color: @kibanaBlue3;
+    border-width: 1px;
+    border-radius: 4px;
+  }
+}
+
+.wizard-step-headings{
+  padding-top: 14px;
+
+  .wizard-step-heading {
+    font-size: 1.3em;
+    padding: 6px 18px;
+    margin: 0 2px;
+    background-color: @settings-add-data-wizard-step-heading-complete-background;
+    color: @settings-add-data-wizard-step-heading-complete-color;
+
+    &.active {
+      cursor: default;
+      background-color: @settings-add-data-wizard-step-heading-active-background;
+      color: @settings-add-data-wizard-step-heading-active-color;
+    }
+    &.aheadActive {
+      cursor: default;
+      background-color: @settings-add-data-wizard-step-heading-incomplete-background;
+      color: @settings-add-data-wizard-step-heading-incomplete-color;
+    }
+  }
+
+  &.complete {
+    .wizard-step-heading:not(.active) {
+      cursor: default;
+    }
+  }
+}
+
+.wizard-step-title {
+  padding-bottom: 14px;
+}
+
+.wizard-nav-buttons {
+  display: flex;
+  float: none;
+  margin-top: 14px;
+
+  div {
+    width:100px;
+    flex: 1 1 auto;
+    display: flex;
+  }
+
+  :nth-child(1) {
+    justify-content: flex-start;
+  }
+  :nth-child(2) {
+    justify-content: center;
+  }
+  :nth-child(3) {
+    justify-content: flex-end;
+  }
+}
+
+pre {
+  border: none;
+  background-color: @settings-filebeat-wizard-panel-bg;
+  border-radius: 0px;
+  white-space: pre-wrap;
+  margin: 0px;
+}
+
+.paste-samples {
+  textarea {
+    width: 100%;
+    height: 250px;
```

Comment:
@alt74 for reference, here's how 250px and 450px look, respectively. To me 450px looks a bit odd here since it starts out as a giant empty box.

![screen shot 2016-04-29 at 2 50 38 pm](https://cloud.githubusercontent.com/assets/6239176/14926401/05bb44a4-0e1a-11e6-9fe5-816bafa503cd.png)

![screen shot 2016-04-29 at 2 51 19 pm](https://cloud.githubusercontent.com/assets/6239176/14926405/0b6ef030-0e1a-11e6-8dcf-ae01bcb6df9b.png)